go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/keploy/go-sdk
 
 go 1.17
 
+replace go.keploy.io/server => ../keploy/
```

Comment:
Please remove or comment this statement.

Reply:
@re-Tick I was planning to remove the statement after the PR was ready to merge, as this allows the reviewers to test the changes without any problems. I have tried building this on my system, it builds without any errors.